src/EvalGraph.purs

```diff
@@ -239,6 +232,27 @@ graphGC' { g, graph_fwd, graph_bwd, inα, outα } =
            select𝔹s inα (vertices g') × g'
    }
 
+dual
```

Comment:
Can we lose this and instead define a way of projecting the result of `graphGC` to a GaloisConnection that we can then apply `dual` to? (Given that we don’t need a version that returns the graph for the tests.)